src/index.js

```diff
@@ -191,8 +191,6 @@ export {
   OrderedList,
   OverflowMenu,
   OverflowMenuItem,
-  PageSelector,
-  Unstable_Pagination, // eslint-disable-line camelcase
```

Comment:
Does this have a new component that we can import or is the functionality now built into regular pagination?

Reply:
whoops this is a good catch, thanks will push the fix @davidicus